libr/core/cconfig.c

```diff
@@ -2169,6 +2162,9 @@ static int cb_searchin(void *user, void *data) {
 		}
 		return false;
 	}
+	if (!strncmp (node->name, "anal", sizeof("anal"))) {
+		core->anal->opt.noncode = strncmp(node->value, "io.maps.x", sizeof("io.maps.x"))? true: false;
```

Comment:
add spaces before (, also dont use sizeof when you want to use strlen.

btw, the ternary operator here is unnecessary, because true/false is implicit if you do == 0 or != 0 .

btw io.maps.rx should be a valid name here and that wont be working with that line

Reply:
oops , yeah `sizeof` was stupid , will fix it 

i will work on this in different PR , i will first try to make tarvis grin with these changes !

Reply:
just push -f @sivaramaaa